src/transformers/modeling_lxmert.py

```diff
@@ -714,28 +817,49 @@ def get_input_embeddings(self):
     def set_input_embeddings(self, new_embeddings):
         self.embeddings.word_embeddings = new_embeddings
 
-    def _prune_heads(self, *args, **kwargs):
-        pass
```

Comment:
This would potentially prevent from having a `NotImplementedError`, instead silently passing.